code/modules/mod/modules/modules_general.dm

```diff
@@ -658,3 +658,108 @@
 		has_tk = FALSE
 		return
 	mod.wearer.dna.remove_mutation(TK)
+
+//Special module for the Captain's Magnate suit, and the CC Corporate suit.
+//Literally only a thing so that https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real
```

Comment:
make this follow the format of the other modules in the file, an autodoc comment with a name and short description

Reply:
? the other mods in the file are literally just 
//"Name"

Reply:
Ah, its not merged yet.
I'll do so